README.md

```diff
@@ -22,7 +22,7 @@ The following authproc filter configuration options are supported:
 * `replace`: Optional, a boolean to use as flag, to replace `verifiedEmailAttribute` if exists. Defaults to `false`.
 * `idpEntityIdIncludeList`: Optional, an array of strings that contains entityIDs for which the module will generate the `verifiedEmailAttribute` attribute. Defaults to empty array.
 * `scopeChecking`: Optional, a boolean to use as flag, that indicates if scopes will be checked, in order to verify an email. Defaults to `false`.
-* `schacHomeOrganization`: Optional, a string to use as the name of the attribute that
+* `schacHomeOrganizationAttribute`: Optional, a string to use as the name of the attribute that
 holds schacHomeOrganization. Defaults to 'schacHomeOrganization'.
```

Comment:
"holds schacHomeOrganization. Defaults to" --> "holds home organisation information. Defaults to"